tests/README

```diff
@@ -34,3 +34,12 @@ See
 
 for more options. But be aware some (like -j) may not work properly with
 the test suite unless a very restricted set of test is run.
+
+
+To test the pythondistdeps.py script, use
```

Comment:
What dependencies do i need? pytest, pyyaml and pip?

Reply:
Plus setuptools which is a dependency of pythondistdeps themselves, added.